packages/manager/modules/exchange/src/domain/add/domain-add.controller.js

```diff
@@ -156,6 +157,9 @@ export default class ExchangeAddDomainController {
 
       delete this.model.attachOrganization2010;
     }
+    const isOvhDomain = this.model.domainType === this.OVH_DOMAIN;
+    this.model.autoEnableDKIM = this.model.autoEnableDKIM && isOvhDomain;
```

Comment:
**question:** What is the expected behavior here?
If `this.model.autoEnableDKIM` is false, the expression always evaluates to false. and do we need to consider `isOvhDomain` only is `this.model.autoEnableDKIM` is true?

Reply:
`this.model.autoEnableDKIM `can be `true` or `false` if  `isOvhDomain=true` and `this.model.autoEnableDKIM` has to be `false` if `sOvhDomain=false`.
Because of the wizzard navigation, model is cleaned just before validation